inc/rom.h

```diff
@@ -130,8 +130,8 @@
 
 #define SAVE_SIZE_SRAM 		32768
 #define SAVE_SIZE_SRAM96 	131072 //TODO: or should this be 98304
```

Comment:
The correct size is 131072. Please remove the comment and I will commit...

Reply:
Alright done, thanks for the info.